app/pages/project/classify.jsx

```diff
@@ -119,11 +119,13 @@ export class ProjectClassifyPage extends React.Component {
     const { actions, project, workflow } = this.props;
 
     this.maybePromptWorkflowAssignmentDialog(this.props);
-    actions.classifier.fetchSubjects(workflow)
-    .then(() => actions.classifier.createClassification(project))
-    .catch((error) => {
-      this.setState({ rejected: { classification: error }});
-    });
+    if (workflow) {
```

Comment:
Should this if wrap `maybePromptWorkflowAssignment` too?

Reply:
Yeah probably, but I think more about that in the separate PR for the refactors to workflow assignment.